### PR TITLE
DPC-556 allows clients to set the _outputFormat parameter on a /Group/$export request to ndjson or application/json. 

### PR DESCRIPTION
**Pull request by richbraman**
### Proposed Changes and ### Change Details

This change allows clients to set the _outputFormat parameter on a /Group/$export request to ndjson or application/ndjson.  Currently only application/fhir+json is supported

### Security Implications

No
- [ ] new software dependencies
n/a

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

- [ ] no PHI/PII is affected by this change

### Acceptance Validation

no, this change should be tested via end to end tests and/or integration of BDT into the ci build .

### Feedback Requested

how to test this appropriately in the dpc build pipeline.

### CODE REVIEW

File: dpc-api/src/main/java/gov/cms/dpc/api/resources/v1/GroupResource.java [hunk 1]
```diff
@@ -351,10 +353,10 @@ private OffsetDateTime handleSinceQueryParam(String since) {
...
      * @param outputFormat param to check
      */
-    private static void checkExportRequest(String outputFormat, String headerPrefer) {
-        // _outputFormat only supports FHIR_NDJSON
-        if (StringUtils.isNotEmpty(outputFormat) && !FHIR_NDJSON.equals(outputFormat)) {
-            throw new BadRequestException("'_outputFormat' query parameter must be 'application/fhir+ndjson'");
+    private static void checkExportRequest(String outputFormat String headerPrefer) {
+        // _outputFormat only supports FHIR_NDJSON, APPLICATION_NDJON, or NDJSON
+        if (StringUtils.isNotEmpty(outputFormat) && (!FHIR_NDJSON.equals(outputFormat) && !APPLICATION_NDJSON.equals(outputFormat) && !NDJSON.equals(outputFormat) )) {
+            throw new BadRequestException("'_outputFormat' query parameter must be 'application/fhir+ndjson','application/ndjson', or 'ndjson' ");
```
Comment (embh) [inline dpc-api/src/main/java/gov/cms/dpc/api/resources/v1/GroupResource.java:359]:
```suggestion
        // _outputFormat only supports FHIR_NDJSON, APPLICATION_NDJSON, or NDJSON
        if (StringUtils.isNotEmpty(outputFormat) && (!FHIR_NDJSON.equals(outputFormat) && !APPLICATION_NDJSON.equals(outputFormat) && !NDJSON.equals(outputFormat) )) {
            throw new BadRequestException("'_outputFormat' query parameter must be 'application/fhir+ndjson', 'application/ndjson', or 'ndjson' ");
```

Comment (richbraman) [reply dpc-api/src/main/java/gov/cms/dpc/api/resources/v1/GroupResource.java:359]: is the constant wrong in the declaration

File: dpc-api/src/main/java/gov/cms/dpc/api/resources/v1/GroupResource.java [hunk 2]
```diff
@@ -369,8 +368,8 @@ private static void checkExportRequest(String outputFormat, List<Pair<String, St
 
     private static void checkOutputFormat(String outputFormat) {
         // _outputFormat only supports FHIR_NDJSON
-        if (StringUtils.isNotEmpty(outputFormat) && !FHIR_NDJSON.equals(outputFormat)) {
-            throw new BadRequestException("'_outputFormat' query parameter must be 'application/fhir+ndjson'");
+        if (!StringUtils.equalsAny(outputFormat, FHIR_NDJSON, APPLICATION_NDJSON, NDJSON)) {
```
Comment (SalGutierrez) [inline dpc-api/src/main/java/gov/cms/dpc/api/resources/v1/GroupResource.java:371]: Should they be case insensitive?

**Review by SalGutierrez — DISMISSED** [review]

File: dpc-api/src/main/java/gov/cms/dpc/api/resources/v1/GroupResource.java (hunk 2 above)
Comment (jonfulk) [reply dpc-api/src/main/java/gov/cms/dpc/api/resources/v1/GroupResource.java:371]: good point... i think they should. Same is true for the `checkRequestHeaders` method I merged in yesterday.

**Review by SalGutierrez — APPROVED** [review]
LGTM

**Review by jonfulk — APPROVED** [review]
LGTM!

**Pull request merged**